src/specification.js

```diff
@@ -712,10 +719,16 @@ var Specification = extend(null, /** @lends Specification# */ {
             var targetUnit = conversion.targetUnit;
             var unitFamily = sourceUnit && sourceUnit.unitFamily;
 
+            var hasPower = _.isNumber(conversion.power);
             var hasFactor = _.isNumber(conversion.factor);
             var hasOffset = _.isNumber(conversion.offset);
-            var autoConvert = !hasFactor && !hasOffset;
+            var autoConvert = !hasFactor && !hasOffset && !hasPower;
 
+            if (hasPower) {
+                if (rawValue !== 0) {
```

Comment:
Please remove that `if` and adapt tests. The result of `Math.pow(0, 0)` is 1 and not 0.